course1/course_en.md

```diff
@@ -0,0 +1,193 @@
+---
+marp: true
+math: mathjax
+paginate: true
+backgroundImage: url('../pics/background_moonbit.png')
+style: |
+    ol > li {
+        list-style-type:decimal
+    }
+---
+
+![bg cover](../pics/course1_front.png)
```

Comment:
Since the cover was designed for the Chinese course, we should just use something simple as other courses
```suggestion
# Modern Programming Ideology

## Course Introduction & Program Design

### Hongbo Zhang
```